docs/docs/helpers.tsx

```diff
@@ -129,16 +130,20 @@ export const WithDynamicPropsExample = () => {
         renderRightControl={renderRightControl}
         renderLeftControl={renderLeftControl}
       />
-      <label>Tiles to show: </label>
-      <button onClick={() => setState({ tilesToShow: state.tilesToShow - 1 })}>-</button>
-      <input value={state.tilesToShow} style={{ width: 40, textAlign: 'center' }} />
-      <button onClick={() => setState({ tilesToShow: state.tilesToShow + 1 })}>+</button>
+      <label htmlFor="input">Tiles to show: </label>
+      <button type="button" onClick={() => setState({ tilesToShow: state.tilesToShow - 1 })}>
+        -
```

Comment:
Minor suggestion to make it readable that it contains no white spaces:

```suggestion
        {'-'}
```